metricbeat/docs/fields.asciidoc

```diff
@@ -23687,7 +23687,7 @@ type: long
 [float]
 === parse
 
-Parses statistic information that occured in the current session
+Parses statistic information that occurred in the current session
```

Comment:
This is an autogenerated file, it shouldn't be manually edited. It looks like you already edited the source file in the oracle module. Just run `make update` to update this file.

Reply:
The travis failure seems to suggest that _something_ went wrong with the docs. Just run `make update fmt` and try again.